views/layout.pug

```diff
@@ -0,0 +1,58 @@
+doctype html
+
+html
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible" content="IE-edge, chrome=1")
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+		link(rel="stylesheet", href="/css/theme.css")
+	body
+		.svg-sprite(style="position: absolute; width: 0; height: 0;")
+			svg.editor-icons(xmlns="http://www.w3.org/2000/svg" style="position: absolute; width: 0; height: 0;")
```

Comment:
the class name here should be app-icons rather than editor icons which is leftover from my markup of the forum components